iotdb-core/node-commons/src/test/java/org/apache/iotdb/commons/path/PatternDFATest.java

```diff
@@ -119,6 +120,55 @@ public void printFASketch2() throws IllegalPathException {
     dfaGraph.print(transitionMap);
   }
 
+  @Test
+  @Ignore
```

Comment:
Why ignore?

Reply:
Sorry, this was my mistake! Thank you for the reminder, I will close this pull request. 😭